ezflow/functional/data_augmentation/operations.py

```diff
@@ -186,8 +188,13 @@ def spatial_transform(
             img2 = img2[::-1, :]
             flow = flow[::-1, :] * [1.0, -1.0]
 
-    y0 = np.random.randint(0, img1.shape[0] - crop_size[0])
-    x0 = np.random.randint(0, img1.shape[1] - crop_size[1])
+    if crop_type.lower() == "center":
+        y0 = H // 2
```

Comment:
I think this would crop the bottom right portion of the image (from the center  [H//2,W//2] to the bottom right [H, W]). Instead, we would like to crop the center portion of the image. 

`y0 = H // 2 - crop_size[0]//2`
`x0 = W // 2 - crop_size[1]//2`

Reply:
Yeah, you're right. Thanks for pointing this out.
Although, when both (H, W) and crop_size are odd, this may not lead to the exact center, right?
I've modified it to something which I think will lead to accurate center every time.